src/parse.rs

```diff
@@ -110,6 +112,15 @@ pub struct Bytes<'a> {
     line: usize,
 }
 
+#[cfg(feature = "integer128")]
+pub(crate) type UInt = u128;
```

Comment:
Could we perhaps call these typedefs differently? Like `LargeUint` or something to indicate they aren't regular integers

Reply:
Done